nemo/backends/pytorch/nm.py

```diff
@@ -328,6 +329,11 @@ def num_workers(self):
     #    """ Property setting the number of workers. """
     #    self._num_workers = nw
 
+    @property
+    def pin_memory(self):
+        """ Property returning the shuffle flag. """
+        return self._pin_memory
```

Comment:
the description is not really matching the "flag" ;)

Reply:
fixed